Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -51,8 +51,12 @@ const trendChannelsStep: number = eventChannelsStep + 1;
 const assetStep: number = trendChannelsStep + 1;
 const connectionStep: number = assetStep + 1;
 const additionalFieldMeterStep: number = connectionStep + 1;
+/* Also undo external turn on in addl
```

Comment:
Is the comment still outstanding?

Reply:
Yes, we need to fix elsewhere before we can turn this step back on